### PR TITLE
Bug fix: Mixed state is inconsistent

### PR DESCRIPTION
**Pull request by shaoshing**
Bug fix for #1776, handle the following cases:

* select multiple layer with different fill color: style panel shows mixed color (used to show black color)
* select multiple layer with different stroke color: style panel shows mixed stoke color (used to show the stroke color of the first layer)

### CODE REVIEW

**Comment by shaoshing** [comment]
Also added `Color#toTinyColor`, no more 

```
var tinycolor = require("tinycolor");
tinycolor(color.toJS());
```

File: src/js/models/color.js [hunk 1]
```diff
@@ -180,11 +181,11 @@ define(function (require, exports, module) {
...
     /**
-     * Return string representation in CSS RGB format. 
-     * @return {string} Example: rgb(100, 100, 0)
+     * Return instance of TinyColor. Provide convenicen for accessing TinyColor functions
+     * @return {TinyColor} 
      */
-    Color.prototype.toCssRGB = function () {
-        return "rgb(" + [Math.round(this.r), Math.round(this.g), Math.round(this.b)].join(", ") + ")";
+    Color.prototype.toTinyColor = function () {
+        return tinycolor(this.toJS());
```
Comment (volfied) [inline src/js/models/color.js]: Can you confirm that tinycolor is doing the rounding for us now?

Comment (volfied) [inline src/js/models/color.js]: Nvm, tinycolor sources say that they first make a percentage out of the color value and then multiply by 255, so they would be rounding in there. :+1:

**Comment by shaoshing** [comment]
Thanks @volfied

**Pull request merged**